packages/react-component-library/src/components/Timeline/__tests__/TodayMarker.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render, RenderResult, cleanup, wait } from '@testing-library/react'
+
+import { TimelineProvider, TimelineContext } from '../context'
+import { TIMELINE_ACTIONS } from '../context/types'
+import { TodayMarker } from '../TodayMarker'
+
+describe('TodayMarker', () => {
+  let wrapper: RenderResult
+
+  describe('today is within range', () => {
+    beforeEach(() => {
+      cleanup()
```

Comment:
It feels like the use of `cleanup` is not required in these tests. When I have seen this I start to think that maybe something else is going wrong.